templates/managed/kafka.html

```diff
@@ -154,151 +163,182 @@ <h3>Cloud architecture</h3>
-      </tr>
-      <tr>
-        <td><strong>REST API</strong></td>
-        <td colspan="2">Yes, administrate Kafka without accessing the underlying Kafka client</td>
-      </tr>
-      <tr>
-        <td><strong>Direct</strong></td>
-        <td colspan="2">Hostname and IP visibility, to connect with your other cloud infrastructure</td>
-      </tr>
-    </table>
+      <h3>Interoperability</h3>
+      <table>
+        <tr>
+          <td><strong>Connectors</strong></td>
+          <td colspan="2">Yes, open source Kafka Connectors for applications</td>
+        </tr>
+        <tr>
+          <td><strong>REST API</strong></td>
+          <td colspan="2">Yes, administrate Kafka without accessing the underlying Kafka client</td>
+        </tr>
+        <tr>
+          <td><strong>Direct</strong></td>
+          <td colspan="2">Hostname and IP visibility, to connect with your other cloud infrastructure</td>
+        </tr>
+      </table>
 
-    <h3>Security and reliability</h3>
-    <table>
-      <tr>
-        <td><strong>Security Patches</strong></td>
-        <td colspan="2">Yes. Canonical provides all security patches.</td>
-      </tr>
-      <tr>
-        <td><strong>Kernel</strong></td>
-        <td colspan="2">Kernel Livepatch</td>
-      </tr>
-      <tr>
-        <td><strong>24/7 Monitoring</strong></td>
-        <td colspan="2">Yes &mdash; Canonical&rsquo;s Kafka experts cover all geographic regions and time-zones</td>
-      </tr>
-      <tr>
-        <td><strong>Break/Fix response</strong></td>
-        <td colspan="2">Yes</td>
-      </tr>
-      <tr>
-        <td><strong>SLA</strong></td>
-        <td colspan="2">Yes, 99.9% up-time.</td>
-      </tr>
-      <tr>
-        <td><strong>ISO270001/2 certified</strong></td>
-        <td colspan="2">Yes</td>
-      </tr>
-      <tr>
-        <td><strong>SOC2 Type 2 certified</strong></td>
-        <td colspan="2">Yes</td>
-      </tr>
-      <tr>
-        <td><strong>GDPR Compliance tested</strong></td>
-        <td colspan="2">Yes</td>
-      </tr>
-    </table>
+      <h3>Security and reliability</h3>
+      <table>
+        <tr>
+          <td><strong>Security Patches</strong></td>
+          <td colspan="2">Yes. Canonical provides all security patches.</td>
+        </tr>
+        <tr>
+          <td><strong>Kernel</strong></td>
+          <td colspan="2">Kernel Livepatch</td>
+        </tr>
+        <tr>
+          <td><strong>24/7 Monitoring</strong></td>
+          <td colspan="2">Yes &mdash; Canonical&rsquo;s Kafka experts cover all geographic regions and time-zones</td>
+        </tr>
+        <tr>
+          <td><strong>Break/Fix response</strong></td>
+          <td colspan="2">Yes</td>
+        </tr>
+        <tr>
+          <td><strong>SLA</strong></td>
+          <td colspan="2">Yes, 99.9% up-time.<br />
+            We can support your specific requirements - <a href="/managed/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>.</td>
```

Comment:
Here too

Reply:
fixed

Reply:
Sorry, but I think they should be lowercased.

Reply:
done.